<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vocabulary</class>
 <widget class="QWidget" name="vocabulary">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расширенный поиск</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Просто наберите ваши выборки разделенные посредством ENTER&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Выбор</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>20</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_2"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEdit_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Язык репертория</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Очистить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Поиск</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="listView">
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>vocabulary</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>plainTextEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>551</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>vocabulary</receiver>
   <slot>filter(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>vocabulary</receiver>
   <slot>selectedModelItem(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>vocabulary</receiver>
   <slot>selectedModelItem(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>plainTextEdit_2</sender>
   <signal>textChanged()</signal>
   <receiver>vocabulary</receiver>
   <slot>changedPlainText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>vocabulary</receiver>
   <slot>openResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>filter(QString)</slot>
  <slot>selectedModelItem(QModelIndex)</slot>
  <slot>changedPlainText()</slot>
  <slot>openResults()</slot>
  <slot>sendOn()</slot>
 </slots>
</ui>
